example/js/api/diplomat-wasm.mjs

```diff
@@ -26,12 +26,20 @@ const imports = {
   }
 }
 
-if (typeof process !== 'undefined') { // Node
-  const fs = await import("fs");
+if (globalThis.process?.getBuiltinModule) {
+  // Node (>=22)
+  const fs = globalThis.process.getBuiltinModule('fs');
```

Comment:
is this more efficient than `await import('fs')`? This seems to be the only functional change

Reply:
My understanding is it is the same as `import('fs')` except it is synchronous and only works for built-in modules.